subscriptions/management/commands/add_prepend_next_to_subscriptions.py

```diff
@@ -0,0 +1,51 @@
+from django.core.management.base import BaseCommand
+
+from subscriptions.models import Subscription
+
+
+class Command(BaseCommand):
+    help = ("A command to add subscription metadata in order to "
+            " inform subscribers of new or changed features.")
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--audio-file',
+            dest="audio_file",
+            type=str,
+            help='A path to the audio file containing the "\
+            "notification message'
+        )
+
+    def handle(self, *args, **options):
+        audio_file = options['audio_file']
+        if not audio_file:
+            self.warning("audio-file is required.")
+            return
```

Comment:
We can probably just set `required=True` on the parser, instead of doing the validation ourselves.